test/es-module/test-esm-repl-imports.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+const path = require('path');
+const { spawn } = require('child_process');
+
+const child = spawn(process.execPath, [
+  '--interactive',
+], {
+  cwd: path.join(__dirname, '../fixtures/es-modules/pkgimports'),
+});
+
+child.stdin.end(`
+try { require('#test'); process.exit(0); }
+catch (e) { process.exit(-1); }
+`);
```

Comment:
The try/catch seems superfluous.
```suggestion
child.stdin.end('require("#test")\n');
```

Reply:
Without try/catch, the exit code is always `0` no matter the required package exists or not. Am I missing something?

Reply:
-1 wouldn’t be correct tho; exit codes are positive integers, i think?

Reply:
AFAIK `-1` is a perfectly valid exit code – and quite common to notify an error.
@rayw000 you're right, we need a catch if we want to determine test success on exit code.

Reply:
https://stackoverflow.com/a/67219932